api-template-java/swagger/swagger.json

```diff
@@ -2,16 +2,16 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-01-30T23:30:18Z",
-    "title": "{api_deployment_node_title}",
```

Comment:
Changes look good. However, we can't merge this PR without the appropriate changes in build-pack-api  that would search & replace values for these keys.

Ex: https://github.com/tmobile/jazz/blob/apigee-plugin/jenkins-build-pack-api/Jenkinsfile#L347

Reply:
updated